src/components/AnnotationForm/AnnotationForm.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * @flow
+ * @file Wraps a component in an IntlProvider
+ * @author Box
+ */
+
+import React from 'react';
+import noop from 'lodash/noop';
+
+import ApprovalCommentForm from '../../../third-party/components/ApprovalCommentForm';
+import Internationalize from '../Internationalize';
+import withFocus from '../withFocus';
+
+type Props = {
+    createAnnotation: Function,
+    onCancel: Function,
+    language?: string,
+    messages?: StringMap,
+    placeholderText: string
+};
+
+const DUMMY_USER = {};
+const IS_EDITING = true;
+const KEEP_INPUT_OPEN = true;
+
+const AnnotationForm = ({ createAnnotation, onCancel, placeholderText, language, messages: intlMessages }: Props) => (
+    <Internationalize language={language} messages={intlMessages}>
+        <ApprovalCommentForm
+            className='ba-annotation-input-container'
+            user={DUMMY_USER}
```

Comment:
Seems like we should actually update `Comment` component to handle null or undefined users instead so we don't have to do placeholder objects like this -- but this is being done in parallel

Reply:
Yea I want to do that when I move ApprovalCommentForm into BRUI as it's own component. For the time being, the decision was to use duplicated components without changing them :-/